src/_blog/libp2p-and-Ethereum.md

```diff
@@ -0,0 +1,311 @@
+We expect that when the time comes for that, collaboration will pick up steam once more!
+
+Thanks for reading!
+
+# Comparing devp2p and libp2p 🤼
+
+This is an optional section that describes the differences between devp2p and libp2p.
+The [devp2p repo](https://github.com/ethereum/devp2p#relationship-with-libp2p) provides an apt contrast of each project's intended scope and design:
+
+*"devp2p is an integrated system definition that wants to serve Ethereum's needs well (although it may be a good fit for other applications, too) while libp2p is a collection of programming library parts serving no single application in particular.
+That said, both projects are very similar in spirit and devp2p is slowly adopting parts of libp2p as they mature."*
+
+Meaning devp2p was designed specifically to fulfill requirements for Ethereum (more on how devp2p adopted parts of libp2p [in the next section](#Coverging-devp2p-and-libp2p-)).
+In particular, devp2p specifies:
+
+- [Ethereum Node Records] (ENR): a format to share and learn an Ethereum node's IP addresses & ports, and its purpose on the network.
+- [Node Discovery Protocol v5] (discv5): a protocol for the Node Discovery system. The system acts like a database of all live nodes and is used for bootstrapping into & finding peers on the network (using ENRs).
+- [RLPx protocol] (RLPx): a TCP-based transport protocol that has a notion of ["capability messages"](https://github.com/ethereum/devp2p/blob/master/rlpx.md#capability-messaging) used during connection establishment.
+    RLPx is used for authentication, stream multiplexing, and more.
+
+Together these specifications define devp2p as an ***integrated** networking **system*** for Ethereum.
+Each component is a requirement of devp2p.
+
+By contrast, libp2p is a ***modular** networking **framework***.
+This means that many different sorts of libp2p networking stacks can be composed by assembling a wide variety general purpose modules.
+
+libp2p provides the necessary modules to create a distributed peer-to-peer network including modules for transport protocols, stream multiplexers, secure channels and authentication, peer discovery, messaging, NAT traversal, and more.
+They are specified in the [libp2p specification repo](https://github.com/libp2p/specs/).
+
+Modularity provides libp2p with a few advantages:
+- Flexibility
+  - An application can pick and choose a combination of modules and assemble a networking stack suited for its needs.
+- Extensiblity
+  - New modules can be added seamlessly thanks to well-defined interfaces and specifications
+- Reach
+    - Thanks to a wealth plug-and-play options, new types of applications can be built with libp2p.
+    Additionally, libp2p enables connectivity to the browser thanks to modular transport protocols.
+    *To learn more about browser connectivity, vist: [connectivity.libp2p.io](https://connectivity.libp2p.io/)*
+
+A great similarity that both devp2p and libp2p share is interoperability.
```

Comment:
Interoperability in the sense that both are implemented in many languages? Or interoperability in the sense that both devp2p and libp2p can communicate? I am not aware of the latter.